src/main/scala/part2dataframes/DataSources.scala

```diff
@@ -75,7 +75,7 @@ object DataSources extends App {
 
   spark.read
     .schema(stocksSchema)
-    .option("dateFormat", "MMM dd YYYY")
+    .option("dateFormat", "MMM d yyyy")
```

Comment:
Hi @daniel-ciocirlan , besides the yyyy, I found only one d could work when reading the csv file in Spark3. Can you help check and verify?
Thank you!